plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -51,6 +51,7 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.hive.metastore.LockComponentBuilder;
 import org.apache.hadoop.hive.metastore.LockRequestBuilder;
+import org.apache.hadoop.hive.metastore.TableType;
```

Comment:
> This partly reverts https://github.com/trinodb/trino/commit/280fee3038d49c1bc82015bdac29616b592d78ef

How much of the commit is the revert, and how much is new code?
I.e. what should i focus on in the review?

Reply:
I removed the partly keyword